webserver/pkgpkr/pkgpkr/settings.py

```diff
@@ -123,10 +123,6 @@
 # Env keys
 GITHUB_CLIENT_ID = os.environ.get('CLIENT_ID')
```

Comment:
Ignore my previous comment, this is what I meant. However, I just want to know why not just leave the exact credentials here without inputting from OS? We can keep a version of setting.py locally and modify it by ourselves.

Reply:
I think our deployment process should drive how we do this, since we'll want a programmatic way to set secrets without committing into our source tree. cc @vpopil @GeraldNamgal